cmd/eskip/writeclient.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import "github.com/zalando/skipper/eskip"
+import (
+	"errors"
+	etcdclient "github.com/zalando/skipper/etcd"
```

Comment:
I will read you comment as "they are **not** very useful"

actually, somebody else defined this aliases, I only moved them. There is an innkeeper variable in the `main` package in conflict with the package name.